core/error/error_list.h

```diff
@@ -90,9 +90,9 @@ enum Error {
 	ERR_PARSE_ERROR,
 	ERR_BUSY,
 	ERR_SKIP, // (45)
-	ERR_HELP, ///< user requested help!!
 	ERR_BUG, ///< a bug in the software certainly happened, due to a double check failing or unexpected behavior.
 	ERR_PRINTER_ON_FIRE, /// the parallel port printer is engulfed in flames
+	ERR_HELP, ///< user requested help!! See --version and --help. Unexposed.
```

Comment:
This changes the numbering of the above enumerations, possibly breaking compatibility.  You might want to reconsider this change.

Reply:
This is such a miniscule change in compatibility that it wouldn't affect any user normally making use of the ERROR codes as normal. Even if it's somehow noteworthy, this is why it the change is done during beta. Do provide an example if I'm wrong.